sklearn_pandas/dataframe_mapper.py

```diff
@@ -119,6 +120,38 @@ def _build(self):
             self.built_features = self.features
         self.built_default = _build_transformer(self.default)
 
+    @staticmethod
+    def _drop_columns(features):
+        """
+        Return a set of columns to be dropped from dataframe
+        """
+        drop_columns = set()
+        try:
```

Comment:
don't use `try` `catch` over here. Otherwise users won't find why the code failed.